angular.json

```diff
@@ -797,5 +797,8 @@
                 }
             }
         }
+    },
+    "cli": {
+        "analytics": "a023a7a7-330e-4272-8ba5-9c5c0acfd65e"
```

Comment:
Why did you add this?

Reply:
I believe this appeared when I had to do npm install again, possibly even before the switch to angular 14 but after one of the major updates to the main branch. I guess I accidentally added it to the commit. I'll delete these lines in the next commit